kv-storage/src/rocks_db.rs

```diff
@@ -16,9 +16,9 @@ impl From<rocksdb::Error> for super::Error {
     }
 }
 
-#[async_trait]
-impl KVStorage for RocksDB {
-    async fn new(path: &str) -> Result<Self, super::Error>
+impl RocksDB {
```

Comment:
No `#[async_trait]` here? Or does [line 71](https://github.com/postech-dao/simperby/pull/57/files#diff-08ebfb5adf93d8f87f492b7d39335011925204878027dfc8c2bc8a55af2c44dbR71) cover this?

Reply:
Also, the separation here is because `new` and `open` is invariant to KVStorage implementations?
Did I get it right?